examples/guide-supported-types-examples/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 #![allow(unused_variables, dead_code)]
 
-extern crate wasm_bindgen;
+use wasm_bindgen;
```

Comment:
Ditto here and elsewhere. In general, we don't need `use some_crate;` in order to do `some_crate::Something`.